runtime/oti/riscvhelpers.m4

```diff
@@ -28,26 +28,33 @@ define({M},{$2{(}$1{)}})
 
 define({ALen},{8})
 
+dnl
+dnl See J9::RV::PrivateLinkageProperties::PrivateLinkageProperties() in RVPrivateLinkage.cpp
+dnl Following definitions must be kept in sync with private linkage.
+dnl
 define({J9VMTHREAD},{s10})
 define({J9SP},{s11})
+define({J9VTABLEINDEX},{t3})
 
 define({FUNC_LABEL},{$1})
 
-define({DECLARE_PUBLIC},{
-    .globl FUNC_LABEL($1)
-    .type FUNC_LABEL($1),function
-})
+define({DECLARE_PUBLIC},{.globl FUNC_LABEL($1)
+    .type FUNC_LABEL($1),function})
```

Comment:
At first I thought the change was that the `.globl` was removed but it's just harder to see now.

Reply:
Yeah, I understand what you mean.  If I leave it before, then generated `.s` file
would look like:
```
.extern fast_jitNewValue
    
    .text
        .globl jitNewValue
    .type jitNewValue,function
    .align 2
jitNewValue:
    .cfi_startproc
    ...
```
See the indentation of `.globl` line. This is because "first indentation level" comes from expading `START_PROC`
```
define({START_PROC},{
    .text
    DECLARE_PUBLIC($1)
    .align 2
```
and second from expanding `DECLARE_PUBLIC`:
```
define({DECLARE_PUBLIC},{dnl
    .globl FUNC_LABEL($1)
    .type FUNC_LABEL($1),function})
```
For some reason I find it disturbing when debugging the generated code. I'll push an update soon, solving the problem by simply inlining `DECLARE_PUBLIC` to `START_PROC` - in RISC-V it is used only once in `START|_PROC` and once in `c_cInterpreter`.